week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -93,7 +104,11 @@ function isAccessibleByTransportMode() { }
    - Returns the name of the location
      e.g: "Tower Bridge"
 */
-function getLocationName() { }
+function getLocationName(arraysOfLocAndTransport) {
+  let location = arraysOfLocAndTransport.slice(0,1);
+  
+  return location[0];
```

Comment:
This happens when we learn new staffs and think we should use it in whatsoever places :) and forget that it might not necessary as can be solved simpler without them:
What if we simply read the first element of the incoming array: `arraysOfLocAndTransport[0]`